core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -147,8 +143,7 @@
   DefaultBlockWorker(BlockMasterClientPool blockMasterClientPool,
       FileSystemMasterClient fileSystemMasterClient, Sessions sessions, BlockStore blockStore,
       UfsManager ufsManager) {
-    super(Executors
-        .newFixedThreadPool(4, ThreadFactoryUtils.build("block-worker-heartbeat-%d", true)));
+    super(ExecutorServiceFactories.fixedThreadPool("block-worker-executor", 4));
```

Comment:
is this the heartbeat thread or something else?

Reply:
This is worker executor thread-pool. Various heartbeats are running on them.